src/main/java/teammates/logic/core/StudentsLogic.java

```diff
@@ -620,12 +620,9 @@ private StudentEnrollDetails enrollStudent(StudentAttributes validStudentAttribu
         return studentList;
     }
 
-    private String getInvalidityInfoInDuplicatedEmail(String[] linesArray, int duplicateEmailIndex) {
-        return "Same email address as the student in line \"" + linesArray[duplicateEmailIndex + 1] + "\"";
-    }
-
     /**
-     * lines is the enrollment lines entered by the instructor.
+     * Returns the formatted string about enrollment problem for invalid using invalidity info of
+     * {@code student} and sanitized enrollment line {@code line}.
```

Comment:
1. Rename the variable {@code line} because it isn't descriptive. A line simply means `-----------` <- this thing is a line. Something like `userInput` will be more descriptive. Same for the other methods too.

1. Update header comments. 
<pre>
Returns <strike>the formatted string</strike> (the caller doesn't need to know 
that it's formatted or not) a {@code String} containing the invalid 
information of the {@code student} and the corresponding sanitised 
invalid {@code userInput}.
</pre>